UI/app/src/main/java/com/example/pathfinder/Activities/DriverDashboard.java

```diff
@@ -156,9 +202,11 @@ public void onCruiseControlBtn(View view) {
         }
     }
 
-    /**Updates text showing seekBar's progress
-     * If cruise control is enabled, vehicle will drive with speed based on seekBar's progress*/
-    private void seekBarListener(){
+    /**
+     * Updates text showing seekBar's progress
+     * If cruise control is enabled, vehicle will drive with speed based on seekBar's progress
+     */
+    private void seekBarListener() {
         seekBar.setOnSeekBarChangeListener(new SeekBar.OnSeekBarChangeListener() {
```

Comment:
Here is another one line 210, seekbar instance declared in master as "m" in the front. Easy fix as well.

Reply:
Thanks Ediz!
I think now it should be fine =)